drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/src/test/java/org/drools/workbench/screens/guided/dtable/backend/server/indexing/TestGuidedDecisionTableFileIndexer.java

```diff
@@ -43,4 +51,34 @@ public void setResourceTypeDefinition( final GuidedDTableResourceTypeDefinition
         this.type = type;
     }
 
+    @Override
+    protected ProjectDataModelOracle getProjectDataModelOracle(final Path path ) {
+        final ProjectDataModelOracle dmo = new ProjectDataModelOracleImpl();
+        dmo.addProjectModelFields( new HashMap<String, ModelField[]>() {{
+            put( "org.drools.workbench.screens.guided.dtable.backend.server.indexing.classes.Applicant",
```

Comment:
@manstis The string can be replaced with `Applicant.class.getCanonicalName()`. From my point of view it has several advantages:
- will be refactored by the IDE when the class moves or is renamed
- it's shorter
- it's DRY

Please take this as a style recommendation/suggestion. I understand that consistency with other tests in this project is another concern so I'll leave it up to you to decide if this is worth the change.